vespajlib/src/main/java/com/yahoo/tensor/TensorAddress.java

```diff
@@ -163,7 +163,7 @@ public String toString() {
     /** Supports building of a tensor address */
     public static class Builder {
 
-        private Pattern identifierPattern = Pattern.compile("[A-Za-z0-9_]+");
+        private Pattern identifierPattern = Pattern.compile("[-,A-Za-z0-9_@]([A-Z,a-z0-9_@$])*");
```

Comment:
I would suggest renaming this one now, sice we do not want it to be confused with identifiers

Reply:
Good point; done